src/Builders/SearchRequestBuilder.php

```diff
@@ -235,6 +235,12 @@ public function load(array $relations, string $modelClass = null): self
         return $this;
     }
 
+    public function setQueryCallback(callable $callback, string $modelClass = null): self
```

Comment:
Can we rename this method to `refineModels`? I'm afraid the current name can create a confusion, because it is a search request builder after all and the word query is ambiguous here.

We also need to describe the new method [here](https://github.com/babenkoivan/elastic-scout-driver-plus/blob/master/docs/available-methods.md) and add it in the list [here](https://github.com/babenkoivan/elastic-scout-driver-plus#search-parameters).